server/src/routes/pointsFetch.ts

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const path = require("path");
+const app = express();
+import {Response} from "express";
+
+app.get('/api/vi/geojson', (res: Response) => {
```

Comment:
- if its a middleware, you need to add your routes starting from the main file and then using router. look at the other files in the folder to see how routes are done. 
- it's v1 not vi. 
- you need to put the req alongside res.